gnucash/report/locale-specific/us/taxtxf.scm

```diff
@@ -3452,29 +3449,29 @@
                         (gnc:html-markup-p
                          (gnc:html-markup/format
                           (string-append
-                             "Selected Report Options:<BR>"
+                             "Selected Report Options:<BR />"
```

Comment:
While you're changing the html BR notes, can you lowercase them at the same time ?

Reply:
Ok. There are lots of <br> elsewhere which I haven't touched because they're translatable.